pkg/api/install/install.go

```diff
@@ -3,6 +3,8 @@ package install
 import (
 	// we have a strong dependency on kube objects for deployments and scale
 	_ "k8s.io/kubernetes/pkg/api/install"
+	_ "k8s.io/kubernetes/pkg/apis/autoscaling/install"
```

Comment:
if you have these here, why also import them elsewhere that already import this?